src/basic/cgroup-util.c

```diff
@@ -209,9 +209,6 @@ bool cg_kill_supported(void) {
         if (supported >= 0)
                 return supported;
 
-        if (cg_all_unified() <= 0)
-                return (supported = false);
```

Comment:
but maybe a cg_is_ready()/cg_is_available() check here?

Reply:
I think that makes sense. If hypothetically v1 was mounted, then `/sys/fs/cgroup/init.scope/cgroup.kill` is not a reserved name and it might exist. The check is effectively free, since the value is cached, so I'd prefer to have it here.